docs/source/user/subdyn/appendixD.rst

```diff
@@ -7,7 +7,7 @@ This is a list of all possible output parameters for the SubDyn module.
 The names are grouped by meaning, but can be ordered in the OUTPUT
 CHANNELS section of the SubDyn input file as the user sees fit. :math:`M \alpha N \beta`,
 refers to node :math:`\beta` of member :math:`\alpha`, where :math:`\alpha` is a number in the range [1,9] and
-corresponds to row :math:`\alpha` in the MEMBER OUTPUT LIST table (see Section ) and
+corresponds to row :math:`\alpha` in the MEMBER OUTPUT LIST table (see :numref:`_SD_Member_Output`) and
```

Comment:
One minor change: the `_SD_Member_Output` should be `SD_Member_Output` in a `:numref:`.  The leading `_` is prepended by `:numref:` to point to the section titled `_SD_Member_Output`.  Could you remove the leading `_`? 

Thanks again!

Reply:
@andrew-platt Fixed! Let me know if there's anything more to do!

Reply:
Perfect!  Thanks!